frontend/frontend-react/src/components/InstanceController.js

```diff
@@ -371,6 +371,38 @@ export class InstanceController extends Component {
         console.log(error.response.data)
         alert('Cannot load. Re-login.\n' + JSON.stringify(error.response.data, null, 2));
       });
+<<<<<<< HEAD
```

Comment:
@michaelzhang114 for some reason this didn't show up as a conflict... can u just fix this on git?